Formula/sdformat12.rb

```diff
@@ -2,16 +2,10 @@ class Sdformat12 < Formula
   desc "Simulation Description Format"
   homepage "http://sdformat.org"
   url "https://github.com/osrf/sdformat.git", branch: "main"
-  version "11.999.999~1"
+  version "11.999.999~2"
   license "Apache-2.0"
   revision 1
```

Comment:
I would leave the `version` as is and increment the `revision`

Reply:
you don't need to remove the bottle block either if we're going to rebuild them in this PR

Reply:
But is building the bottle a good idea? Could we just leave the unstable formulae being built from source, so we don't need to open PRs to update revisions?

Reply:
it's faster to use a bottle than to build from source, so it's a trade-off

Reply:
My concern is having to open these PRs, because they're not automated (yet) like our nightlies, because we're actively adding features to Fortress which are needed by downstream libraries.

I understood that was the resolution of #1314:

_@scpeters: This would actually be closer to our windows CI than Ubuntu nightlies, since it would build the main branch of unreleased packages from source._

Reply:
ok, I viewed it as a small amount of maintenance to rebuild bottles occasionally to keep fortress CI running faster, but I think we weren't on the same page. The following fortress formulae currently have bottles:

~~~
$ grep -l 'bottle do' $(grep -rlI 'url .* "main"' Formula/ | sort)
Formula//ignition-fuel-tools7.rb
Formula//ignition-msgs8.rb
Formula//ignition-physics5.rb
Formula//ignition-rendering6.rb
Formula//ignition-transport11.rb
Formula//sdformat12.rb
~~~

we don't need to change any version numbers or revisions; we just need to remove the bottle blocks for all these formulae and they will rebuild from source each time

Reply:
Cool, let's go with source builds to see if this allows us to iterate faster. If CI starts to take prohibitively long, we can revisit :+1:

Reply:
removing the other fortress bottles in https://github.com/osrf/homebrew-simulation/pull/1530